src/components/NFTPage/CollectorList/Items.vue

```diff
@@ -2,7 +2,18 @@
   <ul>
     <li v-for="owner in ownerList" :key="owner.displayName">
       <div class="flex items-center justify-between">
-        <LinkV2 :to="`/${owner.id}`">{{ owner.displayName | ellipsis }}</LinkV2>
+        <LinkV2
+          class="flex items-center gap-[8px]"
+          :to="`/${owner.id}`"
+        >
+          <Identity
```

Comment:
should we add `v-if="owner.avatar"`?

Reply:
It is guarded in here
https://github.com/likecoin/liker-land/blob/5f55259f196ef7b82c2f6c0dde7c5b427b0b4616/src/mixins/nft.js#L197